### PR TITLE
feat(constants): add verificationLevels

### PR DESCRIPTION
**Pull request by jsmiith**
**Please describe the changes this PR makes and why it should be merged:**
This adds a parseVerification method to the Util class which is helpful due to the fact that many people want to access the guild's verification level and they want it in the string form - not the number form.

**Status**
- [x] Code changes have been tested against the Discord API, or there are no code changes
- [x] I know how to update typings and have done so, or typings don't need updating

**Semantic versioning classification:**  
- [x] This PR changes the library's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Comment by KyeNormanGill** [comment]
change 
```js
if (level === 0) return 'None';
    if (level === 1) return 'Low';
    if (level === 2) return 'Medium';
    if (level === 3) return '(╯°□°）╯︵ ┻━┻';
    if (level === 4) return '┻━┻ ﾐヽ(ಠ益ಠ)ノ彡┻━┻';
    else return undefined;
```

to 
```js
const index = ['Low', 'Medium' '(╯°□°）╯︵ ┻━┻', '┻━┻ ﾐヽ(ಠ益ಠ)ノ彡┻━┻'].indexOf(level);

return index >= 0 ? index : null;
```

Or at least that's what I would do.

**Comment by vladfrangu** [comment]
@KyeNormanGill that would be redundant cause you're getting basically the same number as before... The array is correct, but it should be `return index[level] || 'Unknown'` imo

**Comment by KyeNormanGill** [comment]
Oh my bad, it's the other way around 😄

**Comment by KyeNormanGill** [comment]
```js
return ['Low', 'Medium', '(╯°□°）╯︵ ┻━┻', '┻━┻ ﾐヽ(ಠ益ಠ)ノ彡┻━┻'][level];
```

**Comment by vladfrangu** [comment]
As @appellation said, it should be a constant...

**Comment by Monbrey** [comment]
You left out None from the last example

**Comment by jsmiith** [comment]
I've taken in everyone's suggestions and cleaned the code. It is now this:
```js
/**
   * Converts a verification level to a string.
   * @param {number} level The verification level to parse.
   * @returns {string} The verification level's name
   */
static parseVerification(level) {
  const rules = ['None', 'Low', 'Medium', '(╯°□°）╯︵ ┻━┻', '┻━┻ ﾐヽ(ಠ益ಠ)ノ彡┻━┻'];
  return rules[level] || 'Unknown';
}
```

**Comment by appellation** [comment]
This should be a constant not a function.

File: src/util/Util.js [hunk 1]
```diff
@@ -119,6 +119,16 @@ class Util {
     return { animated: Boolean(m[1]), name: m[2], id: m[3] };
   }
 
+  /**
+   * Converts a verification level to a string.
+   * @param {number} level The verification level to parse.
+   * @returns {string} The verification level's name
+   */
+  static parseVerification(level) {
+    const rules = ['None', 'Low', 'Medium', '(╯°□°）╯︵ ┻━┻', '┻━┻ ﾐヽ(ಠ益ಠ)ノ彡┻━┻'];
```
Comment (MrJacz) [inline src/util/Util.js]: I feel like the array should be in the Constants file.

File: src/util/Constants.js [hunk 2]
```diff
@@ -463,6 +463,23 @@ exports.Colors = {
...
 };
 
+/**
+ * The value set for the verification levels for a guild:
+ * * None
+ * * Low
+ * * Medium
+ * * (╯°□°）╯︵ ┻━┻
+ * * ┻━┻ ﾐヽ(ಠ益ಠ)ノ彡┻━┻
+ * @typedef {string} VerificationLevels
```
Comment (appellation) [inline src/util/Constants.js]:
```suggestion
 * @typedef {string} VerificationLevel
```

The type should be singular here to be consistent with our other typedefs for constants.

**Pull request merged**